content/en/news/support/announcing-1.4-eol/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Support for Istio 1.4 ends on June 5th, 2020
+subtitle: Support Announcement
+description: Upcoming Istio 1.4 end of life announcement.
+publishdate: 2020-05-05
```

Comment:
The publish date will need to be modified to the actual publish date of 1.6.

Reply:
This has nothing to do with 1.6, its 2 months after 1.5.